### PR TITLE
Rename Spec

### PR DESCRIPTION
**Pull request by OR13**
After this is merged, I propose we rename the github repo so the links work better.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -63,7 +63,7 @@
         // [
         // ],
         // name of the WG
-        // wg:     "Secure Data Storage Working Group",
+        // wg:     "Confidential Storage Working Group",
```
Comment (tplooker) [inline index.html:66]: Working group isn't changing?

Comment (OR13) [reply index.html:66]: I should probably delete the commented stuff.

Comment (OR13) [reply index.html:66]: this is find replace, and I ignored it because it was commented.

**Review by dmitrizagidulin — APPROVED** [review]
:+1:

**Review by tplooker — APPROVED** [review]

**Pull request merged**